packages/api-explorer/src/components/ExploreType/ExploreProperty.tsx

```diff
@@ -147,15 +150,23 @@ export const ExplorePropertyReadOnly: FC<ExplorePropertyProps> = ({
  * @param property to describe
  * @constructor
  */
-export const ExplorePropertyDescription: FC<ExplorePropertyProps> = ({
+const ExplorePropertyDescriptionInternal: FC<ExplorePropertyProps> = ({
+  className,
   property,
-}) => (
-  <>
-    {property.description && (
+}) =>
+  property.description ? (
+    <div className={className}>
```

Comment:
@lukelooker  className isn't required. I don't see a default for it, so does it not matter if it's null/undefined?

Reply:
The `className` is how the `ExplorePropertyDescription` Styled Component below "attaches" to `ExplorePropertyDescriptionInternal`. It will get specified consistently but we don't want to require external users to have to specify it since it'll taken care of by the wrapping SC.

Reply:
Got it. Thanks.